addons/link_tracker/models/link_tracker.py

```diff
@@ -73,6 +73,9 @@ def _compute_count(self):
     @api.depends('code')
     def _compute_short_url(self):
         for tracker in self:
+            if not tracker.id or not tracker.code:
+                tracker.short_url = ''
+                continue
             tracker.short_url = urls.url_join(tracker.short_url_host, '%(code)s' % {'code': tracker.code})
```

Comment:
Issue seems to be somehow short_url_host can be "False" so instead I would do `tracker.short_url_host or ''` and `tracker.code or ''`and avoid weird checks on "record exists"